lib/cqm/converter/utils.rb

```diff
@@ -141,7 +141,7 @@ def self.hds_to_qdm_js(js_dependencies, record, qdm_model_attrs)
     # Adjust improper date times from the cql_qdm_patientapi.
     def self.date_time_adjuster(results)
       if results.is_a?(Hash) && results.key?('year') && results.key?('minute')
-        DateTime.new(results['year'], results['month'], results['day'], results['hour'], results['minute'], results['second'], results['millisecond']).to_s
+        DateTime.new(results['year'], results['month'], results['day'], results['hour'], results['minute'], (results['second'] + (results['millisecond'] / 1000.0))).strftime('%Y-%m-%d %H:%M:%S.%N')
```

Comment:
I believe the format string needs to be `%Y-%m-%d %H:%M:%S.%N%Z` in order to include the timezone offset